amqp-quickstart/pom.xml

```diff
@@ -8,7 +8,7 @@
   <properties>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <surefire-plugin.version>2.22.0</surefire-plugin.version>
-    <quarkus.version>999-SNAPSHOT</quarkus.version>
```

Comment:
Qucisktarts on PR must target the development branch and use 999-SNAPSHOT. The bump to the release version happen during the release process.

Reply:
Thanks. Just realised it when looking at the repo.